communications/github-projects.md

```diff
@@ -0,0 +1,55 @@
+## On Github Projects
+
+Since the [Covid Dash development](https://github.com/EQWorks/flaat-dash/projects), we have been using github projects to keep track of progress, issues, and code-specific discussions due to the fast paced nature of the project.
+
+Repos can have multiple projects. Projects can be specific to one extensive task or the whole repo in general - ideally both.
+
+<img width="800" alt="column" src=https://user-images.githubusercontent.com/53827690/83157518-b1cf8e00-a0d1-11ea-9b2c-7a4019ae5066.png>
+
+Projects are a to-do list: easy to keep track of progress as it shows the percentage between open issues / in progress issues / completed issues.
+
+<img width="800" alt="column" src=https://user-images.githubusercontent.com/53827690/83158139-6964a000-a0d2-11ea-9e37-b3f6edd0b7f4.png>
+
+The project is fully customizable and the user can add or delete columns relevant to the project
+<p align="center">
+    <img width="300" alt="column" src=https://user-images.githubusercontent.com/53827690/83159389-ec3a2a80-a0d3-11ea-89fa-7e112f77747d.png>
+    <img width="310" alt="column" src=https://user-images.githubusercontent.com/53827690/83159809-68cd0900-a0d4-11ea-8a00-d6bb168708bb.png>
+</p>
+
+#### On issues
+An issue is created as a to-do item, assigned to a person and labeled:
+- On the ToDo card, click `+`, add a title to your issue, and click `add` a note.
+- Once the note is created, click on `...` of that new ToDo note and choose `convert to issue`.
+- Click on the title of that issue now and a side card will pop on the right. Choose an assignee, add a description, add labels - multiple labels can be added according to the context of that issue.
+- Move the issue between cards to update its status accordingly: when assigned to someone, move to `in progress`, when PR is up, move to `Review in progress`
+
+![image](https://user-images.githubusercontent.com/53827690/83170748-00d1ef00-a0e3-11ea-8f38-ef07b7873671.gif)
+
+#### On Labels
+most common uses are:
+- bug - of a feature that already exists
+- layout/design - style modifications
+- enhancement - improving an existing feature
+- feature - creating a new functionality
```

Comment:
perhaps make this a bit more vague-toned and suggest these as examples, instead of common uses -- they might not be common for projects or individuals' preferences of nomenclatures, etc.

something like:

#### On Labels

Try to use labels to categorize issues and pull requests. There are template labels provided out of the box for each repository. One can also consider some of our own usage examples below:

- bug - of a feature that already exists
- layout/design - style modifications
- enhancement - improving an existing feature
- feature - creating a new functionality